crates/router/src/core/refunds.rs

```diff
@@ -491,7 +491,7 @@ pub async fn validate_and_create_refund(
                 .set_merchant_id(merchant_account.merchant_id.clone())
                 .set_connector_transaction_id(connecter_transaction_id.to_string())
                 .set_connector(connector)
-                .set_refund_type(enums::RefundType::RegularRefund)
+                .set_refund_type(req.refund_type.unwrap_or_default().foreign_into())
```

Comment:
@jarnura , Is it good to use default when refund type is invalid in request? Or should this be an error?

Reply:
The refund type is an enum, and in that case it code will throw an error while serialization itself.